src/Blog/Controller/Posts.php

```diff
@@ -47,29 +47,47 @@ public function run(): Response
 
         return $response;
     }
+    
+    function random_url(){
+
+        $numb = rand(0, 100); 
+
+        return "https://picsum.photos/".$numb;
+
+    }
+    //random_url();
+
+   
+    
 // poniżej wstawiłam swój kod
     private function getPosts(): array
     {
         $posts = [
             ["id" => "1", 
             "title" => "Jak efektywnie uczyć się programowania?", 
             "preamble" => "tekst 1. Norwegia, urzędowo Królestwo Norwegii – państwo w Europie Północnej będące monarchią konstytucyjną, którego terytorium obejmuje zachodnią i północną część Półwyspu Skandynawskiego, Jan Mayen, Svalbard, Wyspę Bouveta i Lofoty. Ma łączną powierzchnię 385 207 km² i liczy 5 165 802 mieszkańców. Graniczy ze Szwecją niemal na całej długości granicy; znacznie krótsze odcinki oddzielają Norwegię od Finlandii i Rosji. Kraj ma również granicę morską przez cieśninę Skagerrakz  Danią. Stolicą Norwegii jest Oslo. Długa, licząca ponad 20 tys. kilometrów linia brzegowa znana jest z charakterystycznych zatok, tzw. fiordów. Nazwa kraju pochodzi od staronordyckiego nord vegen", 
-            "url" => "",
+           // "url" => $this-> random_url(),
             "date" => "12-08-2020"], 
 
             ["id" => "2", 
             "title" => "Podstawy języka JavaScript", 
             "preamble" => "tekst 2", 
-            "url" => "",
+            //"url" => $this-> random_url(),
             "date" => "13-08-2020"], 
 
             ["id" => "3", 
             "title" => "Podstawy języka PHP", 
             "preamble" => "tekst 3", 
-            "url" => "",
+           // "url" => $this-> random_url(),
             "date" => "14-08-2020"]
         ];
 
+        for($i = 0; $i<count($posts); $i++){
```

Comment:
Możesz to zamienić na foreach :)  Jest czytelniejszy.

Reply:
No wlasnie mentor mowil, ze w tym przypadku moze nie do konca dzialac, wiec lepiej jak jest.
Dziekuje